### ISSUE TITLE
DEAD.

### ISSUE THREAD

**Issue by Spidean** [Xonshiz/CoManga#11]
Im using the app on android, every time I open the app i get an error with the prompt "exceptio occurred,please try again after some time"

**Comment by Xonshiz** [comment]
Yes, because this one's dead. The websites I was using to scrape data from, they're gone. Have to find some other reliable source for this. Until then, this won't work.

**Comment by Spidean** [comment]
When will it work again?

**Comment by Xonshiz** [comment]
When I have a stable source for both, Manga and Comic to read/download from.
